dvc/repo/experiments/queue/local.py

```diff
@@ -260,6 +260,21 @@ def shutdown(self, kill: bool = False):
         else:
             self.celery.control.shutdown()
 
+    def attach(
+        self,
+        rev: str,
+        encoding: Optional[str] = None,
+    ):
+        from dvc.ui import ui
+
+        queue_entry: Optional[QueueEntry] = self.get_queue_entry_by_names(
```

Comment:
we probably need a way to filter out active only or queued only when looking up exps by name (we should only be able to attach to active exps)